TrainingApp/src/screens/main/product-list/product-list.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { View, Text, TouchableHighlight } from 'react-native';
+
+import styles from './styles';
+import { BaseStyles } from '../../../app.styles';
+import Product from '../../common/product/product';
+
+class ProductList extends Component {
+    viewAllGroupItems() {}
+
+    getGroupHeaderJSX() {
```

Comment:
change getGroupHeaderJSX to renderGroupHeaderJSX

Reply:
Renamed as mentioned